graphql/distro_resolver.go

```diff
@@ -214,8 +214,6 @@ func (r *plannerSettingsResolver) Version(ctx context.Context, obj *model.APIPla
 	}
 
 	switch utility.FromStringPtr(obj.Version) {
-	case evergreen.PlannerVersionLegacy:
```

Comment:
The E2E tests are failing because the default case is triggered after removing this case statement. This bug can likely be fixed by adjusting the testdata and verified locally by running Spruce and Cypress.